python/lsst/ts/m2gui/utils.py

```diff
@@ -77,3 +77,37 @@ def set_button(
         button.adjustSize()
 
     return button
+
+
+def create_label(name="", point_size=None, is_bold=False):
```

Comment:
Is there really need to change font size? This will fail miserably on e.g. high def displays, will not scale. It's much better to use what Qt provides (e.g. Frames) to frame controls which belongs together.

Reply:
You don't have any use for point_size (grep for create_label, you will see that it's not called with point_size). Changing point size asks for troubles in any dynamically scaled environment. If you really would like to increase label font size, please use ratio (100 - 100% current size):

```python
    if point_size is not None:
        font.setPointSize(font.pointSize() * font_ratio / 100.0)
```

Reply:
I will test this. Thanks for the suggestion!